libnd4j/blas/CMakeLists.txt

```diff
@@ -209,16 +209,27 @@ if(CUDA_BLAS)
         file(GLOB_RECURSE LOOPS_SOURCES false ../include/loops/impl/*.cpp ../include/loops/*.h)
         file(GLOB_RECURSE LOOPS_SOURCES_CUDA false ../include/loops/*.cu)
 
+        if (HAVE_CUDNN)
+            message("cuDNN included")
+            set(CMAKE_CXX_FLAGS " ${CMAKE_CXX_FLAGS} -DHAVE_CUDNN")
+            set(CMAKE_CUDA_FLAGS " ${CMAKE_CUDA_FLAGS} -DHAVE_CUDNN")
```

Comment:
Please don't add this here. Add it to the config.h.in file so they are visible to JavaCPP as well.

Reply:
You mean like you did it for MKL-DNN, right?